cfgov/jinja2/v1/job-description-page/index.html

```diff
@@ -44,29 +44,18 @@ <h1>{{ page.title }}</h1>
                 <strong>({{ page.ordered_grades | join(', ') }})</strong>
                 {% endif %}
                 ${{ '{:,.0f}'.format( page.salary_min ) }}–${{ '{:,.0f}'.format( page.salary_max ) }}
+                <a class="a-link a-link__jump a-link__icon cf-icon cf-icon__after cf-icon-download pay-grade-link" href="https://www.consumerfinance.gov/documents/4722/2016-2019_cfpb_Pay-Table.pdf">
```

Comment:
Please move the `href` to its own line. Optionally, you could also put each class on its own line. Also, I think it's preferable for us to link directly to the canonical location of a file, rather than having it have to go through redirects. That would be http://files.consumerfinance.gov/f/documents/2016-2019_cfpb_Pay-Table.pdf, in this case.